packages/frontend/src/components/ServerState.jsx

```diff
@@ -13,7 +13,7 @@ export default observer(() => {
         }.svg`)}
         alt="sparkles"
       />
-      {!ui.isMobile ? (
+      {!shrink ? (
```

Comment:
i don't understand these changes to the menu from the mobile header. first screenshot is the current version of ceremony-ui branch, second one is with this update:
<img width="282" alt="Screenshot 2023-10-04 at 12 21 51 PM" src="https://github.com/Unirep/trusted-setup/assets/79813968/9c70aef2-b21e-4111-bcd8-ea666558daaf">
<img width="248" alt="Screenshot 2023-10-04 at 12 20 14 PM" src="https://github.com/Unirep/trusted-setup/assets/79813968/56c17f8f-3e95-4850-98b3-93a5a8530bce">

Reply:
the conrtibute page content layout/padding looks good on all phone screens i can simulate. 
the portal image positioning works well on most phones except a few are like this when the you have to scroll a little to see the whole content:
<img width="686" alt="Screenshot 2023-10-04 at 12 11 24 PM" src="https://github.com/Unirep/trusted-setup/assets/79813968/39b17e0b-083b-4db7-8b10-96f163820583">
i think it may be better to have the background image relative rather than fixed, it shouldn't look very different bc the page only needs to be scrolled a small amount, or not at all on many phones.

and the tablet view looks like this:
<img width="406" alt="Screenshot 2023-10-04 at 12 09 44 PM" src="https://github.com/Unirep/trusted-setup/assets/79813968/0bd2cf8b-dc22-4fac-ac86-9afa6e04523f">

also the canvas can have a smaller margin or padding at the top for phone view, it would be better to have some more of the text visible underneath the canvas without scrolling.
<img width="248" alt="Screenshot 2023-10-04 at 12 12 25 PM" src="https://github.com/Unirep/trusted-setup/assets/79813968/c742b702-422d-4ff6-a68c-31983060fdec">

Reply:
change to shrink is because under mobile version but on contribute page, it still need to show not only the image, but also the people waiting in queue. But you're right I need to put `shrink` also on the menu, I forgot it.

Reply:
on tablet I don't know where to put the image is better, maybe ask @ChialiT ?

Reply:
the background gradient is extended, please check.